### PR TITLE
refactor(apps/feature): refactor apps/feature

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description
refactor apps/feature

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [x] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apis/web/feature/views.py [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
 from drf_yasg.utils import swagger_auto_schema
-from rest_framework import status, viewsets
+from rest_framework import generics, status
 
 from apigateway.apps.feature.models import UserFeatureFlag
 from apigateway.utils.responses import OKJsonResponse
 
 
-class FeatureFlagViewSet(viewsets.GenericViewSet):
+class FeatureFlagListAPI(generics.ListAPIView):
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/feature/views.py:29]: ListApi

File: src/dashboard/apigateway/apigateway/apis/web/feature/urls.py [hunk 2]
```diff
@@ -18,8 +18,8 @@
 #
 from django.urls import path
 
-from apigateway.apps.feature import views
+from apigateway.apis.web.feature import views
 
 urlpatterns = [
-    path("flags/", views.FeatureFlagViewSet.as_view({"get": "list"}), name="feature.flags"),
+    path("flags/", views.FeatureFlagListAPI.as_view({"get": "list"}), name="feature.flags"),
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/feature/urls.py]: as_view 中不需要参数

**Comment by wklken** [comment]
单测挂了

File: src/dashboard/apigateway/apigateway/apis/web/feature/urls.py (hunk 2 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/feature/urls.py]: Done

File: src/dashboard/apigateway/apigateway/apis/web/feature/views.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/apis/web/feature/views.py]: Done

**Review by zhu327 — APPROVED** [review]

**Review by wklken — APPROVED** [review]

**Pull request merged**